customer_testing.bat

```diff
@@ -0,0 +1,43 @@
+cd "animations"
+flutter test || exit 1
+cd ..
+
+cd "flutter_maps_firestore"
+flutter test || exit 1
+cd ..
+
+cd "isolate_example"
+flutter test || exit 1
+cd ..
+
+cd "jsonexample"
+flutter test || exit 1
+cd ..
+
+cd "place_tracker"
+flutter test || exit 1
+cd ..
+
+cd "platform_design"
+flutter test || exit 1
+cd ..
+
+cd "platform_view_swift"
+flutter test || exit 1
+cd ..
+
+cd "provider_counter"
+flutter test || exit 1
+cd ..
+
+cd "provider_shopper"
+flutter test || exit 1
+cd ..
+
+cd "shrine"
+flutter test || exit 1
+cd ..
+
+cd "veggieseasons"
+flutter test || exit 1

```

Comment:
does this work? (as in, it causes the file to fail if there's anything error?)

Reply:
I don't have convenient access to a non-personal Windows machine, so I plan to test this when I leave work tonight. This is what I came up with after some online research.

Reply:
another option (easier in bash than batch probably) is to just iterate across all the directories indiscriminately and run flutter run in each one.

Reply:
just a fly-by comment, but we might want to consider consolidating these two scripts into a Dart script

Reply:
@Hixie That's how our normal travis script works. With these, I wasn't sure how to reliably iterate over a list of directories in a batch file, so I separated them out. The bash script was done the same way just so they'd be essentially the same script.

@johnpryan Certainly possible. I just did them as dead simple as I could for now.